scss/tech.scss

```diff
@@ -0,0 +1,40 @@
+ @import 'color';
+
+.technoContent {
+  margin: 12px;
+  background: $color-green;
+  height: 300px;
+  width: 175px;
+  padding: 8.5px;
+  position: relative;
+  border-radius: 5px;
+  display: flex;
+  
+  &::before{
+  content:"";
+  height: 300px;
+  bottom: 5px;
+  right: 5px;
+  width: 175px;
+  border: 1px solid $color-green;
+  border-radius: 5px;
+  position: absolute; 
+  }
+}
+
+.text{
+  text-align: center;
+  flex-direction: row;
+  padding: 5px;
+  margin: 110px 0 0 0;
+  right: 38px;
+}
+
+.imgTechno{
+  text-align: center;
+  position: absolute;
+  right: 38px;
+  bottom: 187px;
```

Comment:
This shouldn't be set as absolute position, you can just center it horizontally and space the content vertically